nix/slothy/default.nix

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+{ stdenvNoCC
+, fetchFromGitHub
+, python311
+, pkgs
+}:
+
+let
+  pythonEnv = python311.withPackages (ps: with ps; [
+    ortools
+    sympy
+    pandas
+    unicorn
+  ]);
+
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "slothy-cli";
+  version = "56ceb705cc864859aa2d581e02e584af6e755668";
+
+  src = fetchFromGitHub {
+    owner = "slothy-optimizer";
+    repo = "slothy";
+    rev = version;
+    sha256 = "sha256-bD+ryt9JO3OwApTIz5I+8eI/q6Bp/rqfaWS4ZUAZRTc=";
+  };
+
+  nativeBuildInputs = [ pkgs.makeWrapper ];
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp slothy-cli $out/bin/
```

Comment:
Is that necessary? Can we not just amend the `PATH`?

Reply:
Not sure what you mean. `$out` is the only thing that lives past the build, so we need to copy it at some point.

Reply:
See for example, our hol_light or the litani build - we always copy to some path with `$out`.